usaspending_api/download/helpers/download_annotation_functions.py

```diff
@@ -563,5 +579,21 @@ def subaward_annotations():
             ),
         ),
         "prime_award_latest_action_date_fiscal_year": FiscalYear("award__latest_transaction__action_date"),
+        "prime_award_cfda_numbers_and_titles": Subquery(
+            TransactionFABS.objects.filter(transaction__award_id=OuterRef("award_id"))
+            .values("cfda_number")
+            .annotate(
+                value=ExpressionWrapper(
+                    ConcatAll(F("cfda_number"), Value(": "), F("cfda_title")),
+                    output_field=TextField(),
+                ),
+                sum=Sum(Coalesce("federal_action_obligation", "original_loan_subsidy_cost")),
+            )
+            .order_by("sum")
```

Comment:
Was this sum put there _just_ for determining the sort-order of the CFDAs listed in the row's CFDA column(s)? 

If so, were you able to test out the impact of this extra mini-agg on downloads of really large files covering a lot of awards (esp. those that may have many CFDAs in their transactions?). I think this extra bit of usability (sorting by "predominant" CFDA) should only be added if it has nearly zero impact on performance.

Reply:
Yeah, the AC wants them sorted by which CFDA has the most money associated with it per award. 

I'm running some test with larger data sets right now, but at the moment it looks like there is some performance lag - 41s without the sum v. 50s with the sum.

Reply:
Are the tests against local or a NonProd environment?

Reply:
tested against QAT db but running the API locally